Kitodo/src/test/java/org/kitodo/production/services/data/TemplateServiceIT.java

```diff
@@ -92,12 +92,12 @@ public void shouldCheckForUnreachableTasksWithException() throws Exception {
 
     @Test
     public void shouldHasCompleteTasks() throws Exception {
-        TemplateDTO templateDTO = templateService.findById(1);
-        boolean condition = templateService.hasCompleteTasks(templateDTO.getTasks());
-        assertTrue("Process DTO doesn't have complete tasks!", condition);
+        TemplateInterface template = templateService.findById(1);
+        boolean condition = templateService.hasCompleteTasks(template.getTasks());
+        assertTrue("Process Interface doesn't have complete tasks!", condition);
```

Comment:
Even you only changed from `DTO` to `Interface` inside the message maybe remove even the `Interface` so the error message is more readable like "Process doesn't have complete tasks!" as the used object type is not needed.

Reply:
You're right, that's ugly. I changed it.